README.md

```diff
@@ -114,6 +114,14 @@ Interface | `ng g interface my-new-interface`
 Enum      | `ng g enum my-new-enum`
 Module    | `ng g module my-module`
 
+angular-cli will add reference to `components`, `directives` and `pipes` automatically in the `app.module.ts`. If you need to add this references to another custom module, follow this steps:
+
+1. `ng g module newModule` to create a new module
+2.  you should change directory with `cd newModule` into the `newModule` folder
```

Comment:
This step should be more forgiving, state that the user should ensure that they generate new items under the `newModule` directory. The reason is that `cd`-ing into the dir is not necessary, as they can simply do... `ng g c new-module/newComponent` (note the the dir name would be `new-module` not `newModule`.